gitops/prer/create_gitops_prs.go

```diff
@@ -33,6 +33,10 @@ import (
 	proto "github.com/golang/protobuf/proto"
 )
 
+func init() {
+	log.SetFlags(log.LstdFlags | log.Lshortfile)
```

Comment:
Show log line number to make it easier to debug. We could remove if this is not preferred.